### ISSUE TITLE
PLY 2 VOXEL | For people having trouble with colors on macOS, this is how to solve the issue

### ISSUE THREAD

**Issue by fffiloni** [Zarbuz/FileToVox#52]
For FileToVox 1.14 release
———

1. Download the the source code zip from the release note
2. Unzip
3. Open the folder with Visual Studio Code
4. Install missing dependencies and restore unresolved dependencies if asked : 

<img width="453" alt="Capture d’écran 2022-01-19 à 17 31 48" src="https://user-images.githubusercontent.com/7447007/150173771-10ef05b9-7b4f-4898-9c25-334c0ec11d0b.png">

5. Then in the Terminal menu section, click "Run build task..." :
<img width="252" alt="Capture d’écran 2022-01-19 à 17 34 12" src="https://user-images.githubusercontent.com/7447007/150174139-d2dc0e29-f3f4-4c37-80a0-cdeab9269e50.png">

6. If the build process ran properly, then you should be able to find that file, open it: 
<img width="448" alt="Capture d’écran 2022-01-19 à 17 36 19" src="https://user-images.githubusercontent.com/7447007/150174464-cddd0af1-485e-4d31-90af-fb860cf55a6e.png">

7. Replace this: 

```
{
  "runtimeOptions": {
    "tfm": "net5.0",
    "framework": {
      "name": "Microsoft.NETCore.App",
      "version": "5.0.0"
    }
  }
}
```

by this : 

```
{
  "runtimeOptions": {
    "tfm": "net5.0",
    "framework": {
      "name": "Microsoft.NETCore.App",
      "version": "5.0.0"
    },
    "configProperties": {
      "System.Drawing.EnableUnixSupport": true
   }
  }
}
```

and Save your changes.

8. And it should be done, you can now run the FileToVox.dll in the console with these commands:

```
> cd Path/to/.../FileToVox-1.14/SchematicToVoxCore/bin/Debug/net5.0
> dotnet FileToVox.dll --i path/to/your_ply_file  --o path/to/your_output_vox --gs 256
```

CONSOLE OUTPUT PROOF  WITHOUT ANY ERROR:
```
[INFO] FileToVox v1.14.0.0
[INFO] Author: @Zarbuz. Contact : https://twitter.com/Zarbuz
[INFO] Specified input path: /Path/TO/myply.ply
[INFO] Specified output path: outputtest
[INFO] Specified grid size: 256
[INFO] Specified output path: Path/TO/myoutput
[INFO] Start reading PLY data...
[INFO] Done.       
[INFO] Max X: 34,16353, Y: 22,1879, 25,34819
[INFO] Min X: 15,393215, Y: 0, 15,55219
[INFO] Size X: 18,770313, Y: 22,1879, 9,7960005
[INFO] Started to voxelize data...
[INFO] Done.       
[INFO] Started quantization of all colors ...
[INFO] Done.       
[INFO] Vox Width: 218
[INFO] Vox Length: 114
[INFO] Vox Height: 257
[INFO] Total blocks: 112347
[INFO] Started to write chunks ...
[INFO] Done.
[INFO] Check total blocks after conversion: 112347
[INFO] Done.

```

And Voilà ! 

Hope that will help you folks !

**Comment by fffiloni** [comment]
I made a tiny native mac app that bundles all the resources needed on macOS to work properly without pain or messing with building process that can be scary, simply drag and drop your file and get you voxels generated in the same folder.

If @zarbuz agrees, i will push it to a repo eventually

https://user-images.githubusercontent.com/7447007/150333679-d7345b48-b9e7-4095-9eb0-dcb17c22c4b7.mp4

**Comment by NikolasTun** [comment]
Hi, I am not a coder. I am trying to use FileToVox on Mac to make Voxel arts. Any update on this issue? Is there an easy way to use FileToVox on Mac now?

**Comment by Zarbuz** [comment]
I will publish soon a new version that enable by default this config to avoid to recompile yourself this project

**Comment by NikolasTun** [comment]
Thank you, Zarbuz. Looking forward to it.

**Comment by Zarbuz** [comment]
Fixed in v1.15. I close this issue
